shared/desktop/yarn-helper/index.tsx

```diff
@@ -109,6 +111,23 @@ const decorateInfo = info => {
   return temp
 }
 
+const warnFail = path => () => console.warn(`Error cleaning tscache ${path}, tsc may be inaccurate.`)
+const clearTSCache = () => {
+  const tsOutDir = path.resolve(__dirname, '..', '..', '.tsOuts')
```

Comment:
you can readDirSync and just blow away everything other than README.md

Reply:
`rimraf` supports globs, so we could have it do `.tsOuts/.tsOut*`, wdyt?

Reply:
even better

Reply:
Done, thanks